In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

In [2]:
ratio=pd.read_csv("C:\\Users\\prasa\\Downloads\\BuyerRatio.csv")
ratio.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [3]:
ratio.drop(["Observed Values"], axis =1, inplace=True)
ratio.head()

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [4]:
from scipy.stats import norm
from scipy.stats import chi2_contingency

In [5]:
p_value=stats.chi2_contingency(ratio)
p_value

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

### P-Value (0.6603)> α (0.05), so we can't reject the null Hypothesis.

### It means male-female buyer rations are similar across regions.